frontend/src/components/dashboard/minesHomePage/Dashboard.js

```diff
@@ -189,6 +198,15 @@ export class Dashboard extends Component {
         this.setState({ mineList: true });
       });
     }
+    // set the lat, long, zoom, and blinking to true
```

Comment:
I don't see anything about blinking in the following code? Since it's clear that `setState` is setting lat/long/zoom, I'd suggest removing this comment.

Reply:
done